samples/azure_attestation/ra_client/client.go

```diff
@@ -3,78 +3,54 @@ package main
 import (
 	"crypto/tls"
 	"crypto/x509"
-	"encoding/base64"
-	"encoding/json"
+	"encoding/binary"
 	"errors"
 	"flag"
 	"fmt"
 	"io/ioutil"
 	"net/http"
-	"time"
+	"reflect"
 
-	"gopkg.in/square/go-jose.v2"
-	"gopkg.in/square/go-jose.v2/jwt"
+	"github.com/edgelesssys/ego/attestation"
 )
 
-// issAddr is the URL of the attestation provider
-const issAddr = "https://shareduks.uks.attest.azure.net"
+// serverAddr is the address of the server
+const serverAddr = "localhost:8080"
 
-// privateClaims are some of the private claims of an Azure Attestation token
-type privateClaims struct {
-	SignerID        string `json:"x-ms-sgx-mrsigner"`
-	ProductID       uint   `json:"x-ms-sgx-product-id"`
-	SecurityVersion uint   `json:"x-ms-sgx-svn"`
-	EnclaveHeldData string `json:"x-ms-sgx-ehd"`
-}
+// attestationProviderURL is the URL of the attestation provider
+const attestationProviderURL = "https://shareduks.uks.attest.azure.net"
```

Comment:
Is this URL actually regionally independent usable, or bound to the UK South region in Azure?

And if it is bound, is there any way to automatically determine the URL, or does someone running this sample on, let's say, Europe West need to change the URL manually?

Reply:
Interesting question, I've tested this by replacing the URL with the URL of the Northern Europe Attestation Provider `https://sharedneu.neu.attest.azure.net` and the example run just fine. 

The [MAA documentation](https://docs.microsoft.com/en-us/azure/attestation/basic-concepts#regional-shared-provider) states

> Azure Attestation provides a regional shared provider in every available region. Customers can choose to use the regional shared provider for attestation, or...

To me it sounds like you should you should use the local one. But on the other hand, it shouldn't matter regarding the concept and security of MAA. So it might be okay to just use the UK South Provider for the sample.